src/arrayMethodSort.js

```diff
@@ -5,7 +5,35 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const defaultCompare = function(a, b) {
+      switch (true) {
+        case (`${a}` > `${b}`): return 1;
+        case (`${a}` < `${b}`): return -1;
+        case (`${a}` === `${b}`): return 0;
+        default: throw new Error('Error: defaulCompre');
+      }
```

Comment:
I don't think that switch construction is really needed here. It looks pretty good with standard if's)

Reply:
It's quite good for me. So, let's agree that this is a matter of personal preferences =)